downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -276,32 +287,45 @@ handle secondWord = arrays:get(input, 1);
     }
 
 ```
+
 ##### Kafka Package Updates
- - SecureSocket record is updated.
- - kafka:Producer, kafka:Consumer and kafka:Listener init updated.
+
+ - The `SecureSocket` record is updated.
+
+ - The `kafka:Producer`, `kafka:Consumer`, and `kafka:Listener` inits are updated.
+
 ```ballerina
 kafka:Producer kafkaProducer = check new(kafka:DEFAULT_URL, config);
 kafka:Producer kafkaProducer = check new (bootstrapServers=”localhost:9092”);
 kafka:Producer kafkaProducer = check new(”localhost:9092”);
 // Same for listener and consumer initialization
 ```
+
 ##### NATS Package Updates
- - Client and listener init updated.
+
+ - The client and listener inits are updated.
+
 ```ballerina
 nats:Client client = check new(url=”http://google.com:9090”, ssl=config);
 nats:Client client = check new(nats:DEFAULT_URL);
 nats:Client client = check new(”http://google.com:9090”);
 // Same for listener initialization
 ```
+
 ##### STAN Package Updates
- - Client and listener init updated.
+
+ - The client and listener inits are updated.
+
 ```ballerina
 stan:Client client = check new(url=”http://localhost:9090”);
 nats:Client client = check new(nats:DEFAULT_URL);
 // Same for listener initialization
 ```
+
 ##### RabbitMQ Package Updates
```

Comment:
Please edit as mentioned in the previous suggestions about naming packages.

Reply:
Resolving as per https://github.com/ballerina-platform/ballerina-dev-website/pull/2093#discussion_r606886986.